.github/workflows/update-sdk.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       - name: Update files
         run:  make tools update-atlas-sdk
       - name: Verify Changed files
-        uses: tj-actions/verify-changed-files@6d688963a73d28584e163b6f62cf927a282c4d11
+        uses: tj-actions/verify-changed-files@82a523f60ad6816c35b418520f84629024d70e1e
```

Comment:
@marcosuma asking you because you approved the PR and may have context: is there a specific reason why we ping this action to a commit instead of a release version? I see that the latest release is [v1.16.1](https://github.com/tj-actions/verify-changed-files/releases/tag/v16.1.1)

Reply:
I do, it's a recommendation by @wtrocki   https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1596#discussion_r1382847384

Reply:
Thanks for sharing the comment. The shared article says

> thi
> For security reasons we want to lock usage of third-party actions to certain commit.
> This can be done by linking action to git commit or even forking actions to private repository.
> 
> When pinning actions to SHAs, it is important to keep the SHAs up to date. You can do this by using a dependency management tool, such as Dependabot.

My point is that when using the version number @v16.1.1 we are still pinging an action to a specific commit but we also make it easy to find which changes were added with it.

Reply:
if we say @v16.1.1 do we still get recommendations by dependabot to upgrade? 
I guess his initial comment was "against" my @v16 that would use the latest published version :-)

Reply:
ah I see, Thanks for the clarification! : ) 

> if we say @v16.1.1 do we still get recommendations by dependabot to upgrade?

yes, we do on minor upgrades. I am not sure if it is what we want here. Happy to resolve this

Reply:
versions can be retagged which is the reason some security aware people recommend to pin to commits instead of tags (unless your dependency management can do checksum validations of the dependency)

Reply:
oh, do you mean v16.1.1  could be changed to point to a different commit? that's a good point then

Reply:
>do you mean v16.1.1 could be changed to point to a different commit?

Correct

GH recommends to use `v16` because you get the latest fixes and has the least amount of maintenance. Security researchers say use a commit because a bad actor could re-tag a "good" release with a malicious commit and you would not notice the changes, int he end is what are the risk of convenience vs security